wiki/tests/test_template_filters.py

```diff
@@ -145,12 +145,12 @@ def test_strip_tags(self):
 
         content = """
         <h1>Some dummy</h1> text. <div>Actually</div> I don't what to write,
-        heh. Don't now, <b>maybe</b> I should citate <>Shakespeare<> or Byron.
```

Comment:
Agreeing! I was just wondering if there had been an original intention with this test case.